### DISCUSSION TITLE
Display nVidia driver version

### DISCUSSION THREAD

**Discussion by Dre9872** [fastfetch-cli/fastfetch#1309] [Q&A] [answered]
So I have been trying to get FF to display the nVidia driver version for hours and I cannot work it out. I have googled and looked at loads of examples, even asked chat GPT to help but I can't find the answer. It probably doesn't help that I have no experience in json, and am pretty new to linux.
Everything works as I would like apart from the GPU section has [Discrete} at the end which I don't like, and the big one GPU driver shows GPU Driver: nVidia: {value} and the value should be the result of the command listed in the text section.
This is my config,jsonc

```
{
  "$schema": "https://github.com/fastfetch-cli/fastfetch/raw/dev/doc/json_schema.json",
  "modules": [
    "os",
    "kernel",
    "packages",
    "lm",
    "shell",
    "de",
    "wm",
    "terminal",
    "terminalfont",
    "memory",
    "board",
    {
      "type": "gpu",
      "temp": true
    },
    {
      "type": "command",
      "shell": "/usr/bin/bash", // Your shell path
      "text": "nvidia-smi | grep 'Driver Version' | awk '{print $3}'", // Command to execute
      "key": "GPU Driver", // Text to display
      "format": "nVidia: {$3}"
    },
    {
      "type": "cpu",
      "temp": true
    },
    "disk"
  ]
}
```

**Comment by CarterLi** [comment, marked as answer]
```
{
    "type": "gpu",
    "key": "GPU Driver",
    "format": "{driver}"
}
```

**Reply by Dre9872** [reply]
Really? that easy. So where did you find this information/where should I have looked?

**Reply by CarterLi** [reply]
`fastfetch -c all --format json`, See GPU part

`fastfetch -h gpu-format`
